src/Actors/actors-startup.lisp

```diff
@@ -201,8 +206,7 @@ THE SOFTWARE.
           (make-actor
            (dlambda
             (:print (&rest things-to-print)
-                    (dolist (item things-to-print)
```

Comment:
this was just redundant code. By vectoring to the "new" blind-print, we inherit the print hook too.